### DISCUSSION TITLE
Decidir qué fuente usar y otras convenciones estéticas

### DISCUSSION THREAD

**Discussion by AstorArico72** [AstorArico72/Proyecto-Transversal-1#21] [Ideas]
Me encontré con que al parecer hemos procedido a hacer las vistas sin acordar cómo hacerlas o qué convenciones estéticas seguir.

Y sí, ésto es en respuesta a que alguien decidió usar Comic Sans MS para las vistas.

**Comment by PabloBarrionuevo** [comment]
Los botones se ve que el IDE los deja con esa fuente de manera predeterminada. Me parece algo no tan importante, lo que hay que hacer es terminar el proyecto para presentar. Recordemos que este TP prácticamente lo realizan los profesores, nosotros únicamente lo adaptamos para que funcione con el modelo de SQL elegido.
Igualmente si luego de terminarlo querés retocar todas las vistas, no lo veo mal, mientras no se modifique la funcionalidad de las mismas.
Podríamos hacerlo mañana.
